azure-pipelines.yml

```diff
@@ -1,170 +1,102 @@
-# ASP.NET Core (.NET Framework)
-# Build and test ASP.NET Core projects targeting the full .NET Framework.
-# Add steps that publish symbols, save build artifacts, and more:
-# https://docs.microsoft.com/azure/devops/pipelines/languages/dotnet-core
-
 trigger:
 - development
 
+pr:
+- development
```

Comment:
want to add in master now that we're merging each sprint?

Reply:
I want to get it working in dev before adding master